optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -318,36 +313,29 @@ def forward(
         attention_mask: torch.LongTensor = None,
         **kwargs,
     ) -> BaseModelOutput:
-        self._create_inference_request()
+        self._compile()
 
-        # Check if inputs are c-like, if not - convert them
-        input_ids = _contiguous_helper(np.array(input_ids))
```

Comment:
It looks much clearer now ! Could this also be integrated for `OVModelForCausalLM` from modeling_decoder.py as well ? happy to do it it in a following PR if not

Reply:
I edited it as much as I could. Hopefully it's better now!:)

I am sorry for the late response here. However, now OV is bumped in setup.py and features can be used from the official release.